Client/Application/ClientPreferences.swift

```diff
@@ -218,9 +218,19 @@ extension Preferences {
     }
     
     final class Chromium {
+        /// The boolean determine Bookmark Migration is finished on client side
         static let syncV2BookmarksMigrationCompleted = Option<Bool>(key: "chromium.migration.bookmarks", default: false)
-        static let syncV2BookmarksMigrationCount = Option<Int>(key: "chromium.migration.bookmarks.count", default: 0)
```

Comment:
Are you sure we can delete this key and it is replaced with `syncV2ObjectMigrationCount` but has a different string/key?

Reply:
Yes because syncV2ObjectMigrationCount is the count for number for how many times the migration failed so In out situation after user upgraded their app version, the migration process will work again for the history items.